src/features/wallet/root/WalletView.tsx

```diff
@@ -32,13 +33,25 @@ const WalletView = ({ layout, animationPoints }: Props) => {
 
   const { t } = useTranslation()
 
+  const navigation = useNavigation()
```

Comment:
We had navigation pretty centralized in the old app which seemed to work well. I haven't touched it too much in the new app to see, but Im good with keeping it together if you think that would make more sense

Reply:
I'm ok centralizing it if that's cleaner. We end up needing to put a decent number of screens in the root view because we want the modal to cover the tabs, and that seemed to be the only way to make that work, unless there's another way you know of to avoid stuffing modals in the root nav